src/analysis/liouville/inequalities_and_series.lean

```diff
@@ -0,0 +1,122 @@
+λ x x1 y y1 xy, (one_div_lt_one_div (mem_Ioi.mp y1) (mem_Ioi.mp x1)).mpr xy
+
+lemma one_div_mono_exp (m1 : 1 ≤ m) {a b : ℕ} (ab : a ≤ b) :
+  1 / m ^ b ≤ 1 / m ^ a :=
+begin
+  refine (one_div_le_one_div _ _).mpr (pow_le_pow m1 ab);
+  exact pow_pos (lt_of_lt_of_le zero_lt_one m1) _
+end
+
+lemma one_div_pow_strict_mono (m1 : 1 < m) {a b : ℕ} (ab : a < b) :
+  1 / m ^ b < 1 / m ^ a :=
+begin
+  refine one_div_pow_strict_mono_decr_on _ _ (pow_lt_pow m1 ab);
+  exact pow_pos (zero_lt_one.trans m1) _
+end
+
+section lemmas_about_summability_and_sums
+
+/-- A series whose terms are bounded by the terms of a converging geometric series convergences. -/
+lemma summable_inv_pow_ge {f : ℕ → ℕ} (hm : 1 < m) (fi : ∀ i, i ≤ f i) :
+  summable (λ i, 1 / m ^ f i) :=
+begin
+  refine summable_of_nonneg_of_le
+    (λ a, one_div_nonneg.mpr (pow_nonneg (zero_le_one.trans hm.le) _)) (λ a, _)
+    (summable_geometric_of_lt_1 (one_div_nonneg.mpr (zero_le_one.trans hm.le))
+      ((one_div_lt (zero_lt_one.trans hm) zero_lt_one).mpr (one_div_one.le.trans_lt hm))),
+  rw [div_pow, one_pow],
+  refine (one_div_le_one_div _ _).mpr (pow_le_pow hm.le (fi a));
+  exact pow_pos (zero_lt_one.trans hm) _
+end
+
+end lemmas_about_summability_and_sums
+
+/--  Partial inequality, works with `m ∈ ℝ` satisfying `1 < m`. -/
+lemma calc_liou_one (m1 : 1 < m) (n : ℕ) :
+∑' (i : ℕ), 1 / m ^ (i + (n + 1))! < (1 - 1 / m)⁻¹ * (1 / m ^ (n + 1)!) :=
+-- two useful inequalities
+have m0 : 0 < m := (zero_lt_one.trans m1),
+have mi : abs (1 / m) < 1 :=
+  (le_of_eq (abs_of_pos (one_div_pos.mpr m0))).trans_lt ((div_lt_one m0).mpr m1),
+calc (∑' i, 1 / m ^ (i + (n + 1))!)
+    < ∑' i, 1 / m ^ (i + (n + 1)!) :
+    -- to show the strict inequality between these series, we prove that:
+    tsum_lt_tsum_of_nonneg
+      -- 1. the first series has non-negative terms
+      (λ b, one_div_nonneg.mpr (pow_nonneg m0.le _))
+      -- 2. the second series dominates the first
+      (λ b, one_div_mono_exp m1.le (b.add_factorial_succ_le_factorial_add_succ n))
+      -- 3. the term with index `i = 2` of the first series is strictly smaller than
+      -- the corresponding term of the second series
+      (one_div_pow_strict_mono m1 (n.add_factorial_succ_lt_factorial_add_succ rfl.le))
+      -- 4. the second series is summable, since its terms grow quickly
+      (summable_inv_pow_ge m1 (λ j, nat.le.intro rfl))
+... = ∑' i, (1 / m) ^ i * (1 / m ^ (n + 1)!) :
+    -- split the sum in the exponent and massage
+    by { congr, ext i, rw [pow_add, ← div_div_eq_div_mul, div_eq_mul_one_div, ← one_div_pow i] }
+-- factor the constant `(1 / m ^ (n + 1)!)` out of the series
+... = (∑' i, (1 / m) ^ i) * (1 / m ^ (n + 1)!) : tsum_mul_right
+... = (1 - 1 / m)⁻¹ * (1 / m ^ (n + 1)!) :
+    -- the series if the geometric series
+    mul_eq_mul_right_iff.mpr (or.inl (tsum_geometric_of_abs_lt_1 mi))
+
+lemma calc_liou_two_zero (n : ℕ) (hm : 2 ≤ m) :
```

Comment:
```suggestion
lemma liouville.aux_calc (n : ℕ) (hm : 2 ≤ m) :
```
I agree that this one seems very specialized, and unlikely to benefit from a long name following the convention. But namespacing it makes sure it doesn't pollute the root namespace.